### PR TITLE
Fix file descriptor issues with RxFile#flow

### PR DESCRIPTION
**Pull request by Sammers21**
* Close underlying AsyncFile after read stream completion

### CODE REVIEW

**Comment by paulodamaso** [comment]
@g4s8 Please assign me

**Comment by g4s8** [comment]
@0crat assign me

File: src/test/java/com/artipie/asto/RxFileTest.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ public void rxFileFlowWorks() throws IOException {
             .map(bytes -> new String(new ByteArray(bytes).primitiveBytes()))
             .blockingGet();
         MatcherAssert.assertThat(hello, Matchers.equalTo(content));
-        temp.toFile().deleteOnExit();
+        temp.toFile().delete();
```
Comment (g4s8) [inline src/test/java/com/artipie/asto/RxFileTest.java]: @Sammers21 let's use junit `TempDir` annotation

File: src/test/java/com/artipie/asto/RxFileTest.java [hunk 2]
```diff
@@ -81,6 +81,7 @@ public void rxFileSaveWorks() throws IOException {
                 )
             ).blockingAwait();
         MatcherAssert.assertThat(new String(Files.readAllBytes(temp)), Matchers.equalTo(hello));
+        temp.toFile().delete();
```
Comment (g4s8) [inline src/test/java/com/artipie/asto/RxFileTest.java]: @Sammers21 and here (`TempDir`)

**Comment by g4s8** [comment]
@rultor release, tag=`0.13.3`

**Comment by 0crat** [comment]
Job `gh:artipie/asto#75` is not assigned, can't get performer

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/artipie/asto/issues/75#issuecomment-588227419))

@g4s8 This pull request [#75](https://github.com/artipie/asto/issues/75) is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/90c677fb-b1a2-45fe-91fb-d180d3b191a2); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@beb5dca8c5dc, thread: PQ-CT2E6TK9B
com.zerocracy.claims.ClaimIn[214] java.lang.IllegalArgumentException: Parameter "age" not found in "Order was canceled" among: [reason, job, cause, _parent_sig, flow, voluntarily, login]
```

[0.51.6](https://github.com/zerocracy/farm): CID: [4388fb4e-8f0d-4631-a34d-71a1677b30f2](https://www.0crat.com/footprint/CT2E6TK9B/4388fb4e-8f0d-4631-a34d-71a1677b30f2), Type: "Order was canceled"

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) resigned from [#75](https://github.com/artipie/asto/issues/75), since the job is not in scope anymore

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Pull request merged**